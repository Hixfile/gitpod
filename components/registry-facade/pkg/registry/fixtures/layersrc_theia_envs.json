{
  "sourceRef": "eu.gcr.io/gitpod-core-dev/build/theia-ide:akosyakov-enable-code-cli-2422.12",
  "content": {
    "afe2f8881f672fab816a92bee67ba9e0fd762a3225a0ee4f37a34a0a2f88d8fb": "eyJjcmVhdGVkIjoiMjAyMC0xMi0xOFQxMzo1ODowMS44NTA4MDA5NDdaIiwiYXJjaGl0ZWN0dXJlIjoiYW1kNjQiLCJvcyI6ImxpbnV4IiwiY29uZmlnIjp7IkVudiI6WyJQQVRIPS91c3IvbG9jYWwvc2JpbjovdXNyL2xvY2FsL2JpbjovdXNyL3NiaW46L3Vzci9iaW46L3NiaW46L2JpbiIsIkdJVFBPRF9CVUlMVF9JTl9QTFVHSU5TPS90aGVpYS9ub2RlX21vZHVsZXMvQGdpdHBvZC9naXRwb2QtaWRlL3BsdWdpbnMvIiwiR0lUUE9EX0FQUEVORF9FTlZfUEFUSD0vaWRlL2JpbjoiLCJHSVRQT0RfU0VUX0VOVl9FRElUT1I9Z3Agb3BlbiAtdyIsIkdJVFBPRF9TRVRfRU5WX1ZJU1VBTD1ncCBvcGVuIC13IiwiR0lUUE9EX1NFVF9FTlZfR0lUX0VESVRPUj1ncCBvcGVuIC13Il0sIldvcmtpbmdEaXIiOiIvaWRlLyJ9LCJyb290ZnMiOnsidHlwZSI6ImxheWVycyIsImRpZmZfaWRzIjpbInNoYTI1NjphYTg0MDBlNWM3MDg3ZWQwYzY3NTlmYzEwMzM1ZTM1NDk2YmIxYWQwMDBhMzM5ODYzZDBhYjdmMGVkZWZkYWI3Iiwic2hhMjU2Ojg4NDM4N2ZlZDZiZTZlNzVhNmRiMjQ2NTBmNzVhMzc5YjRiNjdiZjZiYmY4ZDIyOTY2NDVkY2VmYmQ1MjAxODkiLCJzaGEyNTY6NDVmOTg1MDc1Y2NkNTBiMDE0Y2I3MTFkMTBkZjlkNGI1MTgyZWQ3Y2ZmZGE2YjllZGJkZWYwNDlmZWJhOTcwNyIsInNoYTI1NjoyZDI2OTA0ODdiZDcwZWNkNDdlOTA2ZjdiNzkwYWQ1OWVmMWY3OWFmMDViNWJjYzEwNDNmNjFlMjFhMWNlM2UwIiwic2hhMjU2OmMwMzc0NjIyZThiZWQwZGU4OGRhNzM5ZjI5YjY0Y2RkYWQ1M2QxMmFiMTQ4MDU2NGFiNTA2NmI4NDkyNWFhOTYiLCJzaGEyNTY6YjkxMzUyZWY0NTRhM2FmMDU2ZjE1MDIyYWUzNTAzMjZkMjJmN2VhN2U5NzJlY2ZiMWIwYzhkYjUxYjkxY2JiNCIsInNoYTI1NjowYjQ3ZTk3Yzk3YWVjZDM0MDg1MDhjYjhhNGYwMGQwYmM2NzA2YTg4MjA5YTEzYzYwZDA5MTA3NmE5MjI1MTMwIiwic2hhMjU2OjQ3OTE3NGMxODE0OTViMjMyNmNhZTQwMjBkMjg3NzNhOGVjODY5ODFjM2VkMjBhOTRlODgwNTBkNDRmY2I0NWQiXX0sImhpc3RvcnkiOlt7ImNyZWF0ZWQiOiIyMDIwLTEyLTE4VDEzOjQzOjQzLjQwNzAyNTQxMVoiLCJjcmVhdGVkX2J5IjoiL2Jpbi9zaCAtYyAjKG5vcCkgQ09QWSBkaXI6ZTVlODI2MDdhMzU3NzU5OGU0ZGY2Yjg5NjNjMzJhZmE4NDNiZjA2YTMxZTZjYTBjY2U2N2IzZmFiNjJlMGZlMiBpbiAvdGhlaWEvICJ9LHsiY3JlYXRlZCI6IjIwMjAtMTItMThUMTM6NDM6NDUuNjU0NjU1ODJaIiwiY3JlYXRlZF9ieSI6Ii9iaW4vc2ggLWMgIyhub3ApIFdPUktESVIgL2lkZS8ifSx7ImNyZWF0ZWQiOiIyMDIwLTEyLTE4VDEzOjQzOjQ1LjkwNTAxOTIxNFoiLCJjcmVhdGVkX2J5IjoiL2Jpbi9zaCAtYyAjKG5vcCkgQ09QWSBmaWxlOjJhNWY1YWRkYTZiZDEzMmRjZmFlNGFkNjBkYTZmOWYxNmY4ZjM3NjVjNGZjNTViNmI5NDYwYzhiMjc5OThlZGUgaW4gL2lkZS8gIn0seyJjcmVhdGVkIjoiMjAyMC0xMi0xOFQxMzo0Mzo0Ni4yMTkxMjk4MzlaIiwiY3JlYXRlZF9ieSI6Ii9iaW4vc2ggLWMgIyhub3ApICBFTlYgR0lUUE9EX0JVSUxUX0lOX1BMVUdJTlM9L3RoZWlhL25vZGVfbW9kdWxlcy9AZ2l0cG9kL2dpdHBvZC1pZGUvcGx1Z2lucy8iLCJlbXB0eV9sYXllciI6dHJ1ZX0seyJjcmVhdGVkIjoiMjAyMC0xMi0xOFQxMzo0Mzo0Ny40ODg5Mzg1NjlaIiwiY3JlYXRlZF9ieSI6Ii9iaW4vc2ggLWMgIyhub3ApIENPUFkgZGlyOmJiNmYzMTYyNjVkNzc4ODM2MTRjZTk4MDZiZDg0YjNlYjNiYmE3OGYwMzRhMmNiNGQ1OGQ1MzMxZTc4NjhmYzYgaW4gL3RoZWlhL25vZGVfbW9kdWxlcy9AZ2l0cG9kL2dpdHBvZC1pZGUvcGx1Z2lucy8gIn0seyJjcmVhdGVkIjoiMjAyMC0xMi0xOFQxMzo0Mzo0OS4yODMwMDcyNDhaIiwiY3JlYXRlZF9ieSI6Ii9iaW4vc2ggLWMgIyhub3ApIENPUFkgZmlsZTpjOTdhNWQ2M2UyMGM5ODkwNzYxYmQ1MzJlZWNkODIzOTViNTg5ODViODE5NDVhYzllZjViM2MxZTBkOTMxZDU4IGluIC90aGVpYS9zdXBlcnZpc29yICJ9LHsiY3JlYXRlZCI6IjIwMjAtMTItMThUMTM6NDM6NDkuNzc2MzQzNjQ1WiIsImNyZWF0ZWRfYnkiOiIvYmluL3NoIC1jICMobm9wKSBDT1BZIG11bHRpOmJhZTcwYzdhZGYwY2QzYjMyNjgwZGUwNzcxMjYwOGE5Y2E3NTg2ZjI1OTBjMjc4MWFhNzg4MzAxZDA2YzA5YmYgaW4gL3RoZWlhLyAifSx7ImNyZWF0ZWQiOiIyMDIwLTEyLTE4VDEzOjQzOjUwLjA1Mjk0NjU4N1oiLCJjcmVhdGVkX2J5IjoiL2Jpbi9zaCAtYyAjKG5vcCkgQ09QWSBtdWx0aTo0ZDAzNzBmMWEzMjg2MTE0YTY5MWM2YzIzOGNiOTA2YjRmMDUyZGM2ZTczMDhlMzI1YjA5YTVmYzIzNDk2ZTkyIGluIC90aGVpYS8gIn0seyJjcmVhdGVkIjoiMjAyMC0xMi0xOFQxMzo0Mzo1MC4yNjU5MzQyNVoiLCJjcmVhdGVkX2J5IjoiL2Jpbi9zaCAtYyAjKG5vcCkgQ09QWSBkaXI6MmJlNGFjZmE0M2JkMTIwMjgwYzdhYjFlMjU5NzQ5N2U0MWQ4NGZmZmJjNTJlM2YxMmY4YjU2M2Q1YzY5YjRiYSBpbiAvaWRlL2JpbiAifSx7ImNyZWF0ZWQiOiIyMDIwLTEyLTE4VDEzOjQzOjUwLjYzMDM1OTcwNloiLCJjcmVhdGVkX2J5IjoiL2Jpbi9zaCAtYyAjKG5vcCkgIEVOViBHSVRQT0RfQVBQRU5EX0VOVl9QQVRIPS9pZGUvYmluOiIsImVtcHR5X2xheWVyIjp0cnVlfSx7ImNyZWF0ZWQiOiIyMDIwLTEyLTE4VDEzOjQzOjUxLjAyNDg0MzE3MVoiLCJjcmVhdGVkX2J5IjoiL2Jpbi9zaCAtYyAjKG5vcCkgIEVOViBHSVRQT0RfU0VUX0VOVl9FRElUT1I9Z3Agb3BlbiAtdyIsImVtcHR5X2xheWVyIjp0cnVlfSx7ImNyZWF0ZWQiOiIyMDIwLTEyLTE4VDEzOjU4OjAxLjU3OTIxODY5OVoiLCJjcmVhdGVkX2J5IjoiL2Jpbi9zaCAtYyAjKG5vcCkgIEVOViBHSVRQT0RfU0VUX0VOVl9WSVNVQUw9Z3Agb3BlbiAtdyIsImVtcHR5X2xheWVyIjp0cnVlfSx7ImNyZWF0ZWQiOiIyMDIwLTEyLTE4VDEzOjU4OjAxLjg1MDgwMDk0N1oiLCJjcmVhdGVkX2J5IjoiL2Jpbi9zaCAtYyAjKG5vcCkgIEVOViBHSVRQT0RfU0VUX0VOVl9HSVRfRURJVE9SPWdwIG9wZW4gLXciLCJlbXB0eV9sYXllciI6dHJ1ZX1dfQ==",
    "b5d591d9da7f7491a3f0f37474a07b3e6501db73ec11a49e3506fa6d5132fb0f": "eyJzY2hlbWFWZXJzaW9uIjoyLCJjb25maWciOnsibWVkaWFUeXBlIjoiYXBwbGljYXRpb24vdm5kLmRvY2tlci5jb250YWluZXIuaW1hZ2UudjEranNvbiIsImRpZ2VzdCI6InNoYTI1NjphZmUyZjg4ODFmNjcyZmFiODE2YTkyYmVlNjdiYTllMGZkNzYyYTMyMjVhMGVlNGYzN2EzNGEwYTJmODhkOGZiIiwic2l6ZSI6NDE5MH0sImxheWVycyI6W3sibWVkaWFUeXBlIjoiYXBwbGljYXRpb24vdm5kLmRvY2tlci5pbWFnZS5yb290ZnMuZGlmZi50YXIuZ3ppcCIsImRpZ2VzdCI6InNoYTI1NjozYTJmZjBjOGI3YTk0OWJmNDU3OWRmMjJmZDQyY2UwMDg4YTM2NGUzZjY5YTc0NjhhM2MxMjkxOWJlNzkyZWY1Iiwic2l6ZSI6ODYwODYzMDR9LHsibWVkaWFUeXBlIjoiYXBwbGljYXRpb24vdm5kLmRvY2tlci5pbWFnZS5yb290ZnMuZGlmZi50YXIuZ3ppcCIsImRpZ2VzdCI6InNoYTI1Njo3YzE5ZThjOTc0MTE5NDE0ZjM3MGFkNWZmNDk4NTM1OWQ1NGU5YjAyMjRlNWM0MjE3YzI1N2Q3MDI5YzBkMjg0Iiwic2l6ZSI6MTI2fSx7Im1lZGlhVHlwZSI6ImFwcGxpY2F0aW9uL3ZuZC5kb2NrZXIuaW1hZ2Uucm9vdGZzLmRpZmYudGFyLmd6aXAiLCJkaWdlc3QiOiJzaGEyNTY6NTgyMjk2NWNiOTkwM2YxZjEyNzg0ODFiYjYxYjk5ZDA1NjE5YzA5YTA3MmYxN2FiN2JiZDcwZmI1NjRlYjkwOSIsInNpemUiOjI1OH0seyJtZWRpYVR5cGUiOiJhcHBsaWNhdGlvbi92bmQuZG9ja2VyLmltYWdlLnJvb3Rmcy5kaWZmLnRhci5nemlwIiwiZGlnZXN0Ijoic2hhMjU2OmI2MDk4YTZmYzc5MmE3MDZmYjJjMDMwYmVlNjcwZGU1NDc4ZTViOGQxODUzZGI3N2RjMTZiMmNhYTE3Yzk3N2EiLCJzaXplIjoxMjgzODUzOTZ9LHsibWVkaWFUeXBlIjoiYXBwbGljYXRpb24vdm5kLmRvY2tlci5pbWFnZS5yb290ZnMuZGlmZi50YXIuZ3ppcCIsImRpZ2VzdCI6InNoYTI1NjpmYmJlNzkxOTQ4ODVlOGNmZDQ5MjExYjExNmUxMDY5NDg5MDRlNmYwMGEyYTRlMTIwYmE5ZGJjN2QyNzJhM2Q3Iiwic2l6ZSI6NTU3MjQxM30seyJtZWRpYVR5cGUiOiJhcHBsaWNhdGlvbi92bmQuZG9ja2VyLmltYWdlLnJvb3Rmcy5kaWZmLnRhci5nemlwIiwiZGlnZXN0Ijoic2hhMjU2OjdlYjIxNWQ4NWIwNDZjZmM3OWZjNjc2MzlkOGQ5YzY4NDllMDVhNWQ5NmJmMGExOGIwZDQyM2Q3MjU1MjRhNzgiLCJzaXplIjo1MTk1Mzk4fSx7Im1lZGlhVHlwZSI6ImFwcGxpY2F0aW9uL3ZuZC5kb2NrZXIuaW1hZ2Uucm9vdGZzLmRpZmYudGFyLmd6aXAiLCJkaWdlc3QiOiJzaGEyNTY6Njc2NGRiYWJjODZhYmRjOTRhNTllYjY5ZGI5M2M2OWI0MWYyNjhlY2M1ZmM5MTljOTkwYjI0YTdkMmRmZjU0YiIsInNpemUiOjMzNX0seyJtZWRpYVR5cGUiOiJhcHBsaWNhdGlvbi92bmQuZG9ja2VyLmltYWdlLnJvb3Rmcy5kaWZmLnRhci5nemlwIiwiZGlnZXN0Ijoic2hhMjU2OmI3MTlmYTE3NDkxZWFkZTI5NDQ1OGIyYWJjZGQ3NTlkMWE2NDQ1MmRlY2YyMDdlMzgzZjM2MmUyN2EzMGI0MmUiLCJzaXplIjoyMjF9XX0=",
    "eu.gcr.io/gitpod-core-dev/build/theia-ide:akosyakov-enable-code-cli-2422.12": "eyJtZWRpYVR5cGUiOiJhcHBsaWNhdGlvbi92bmQuZG9ja2VyLmRpc3RyaWJ1dGlvbi5tYW5pZmVzdC52Mitqc29uIiwiZGlnZXN0Ijoic2hhMjU2OmI1ZDU5MWQ5ZGE3Zjc0OTFhM2YwZjM3NDc0YTA3YjNlNjUwMWRiNzNlYzExYTQ5ZTM1MDZmYTZkNTEzMmZiMGYiLCJzaXplIjoxOTkyfQ=="
  },
  "envs": [
    "PATH=/usr/bin",
    "EDITOR=vi",
    "GIT_EDITOR=vi"
  ]
}