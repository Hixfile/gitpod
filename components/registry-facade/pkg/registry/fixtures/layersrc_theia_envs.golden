{
    "error": "",
    "layer": [
        {
            "Descriptor": {
                "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                "digest": "sha256:3a2ff0c8b7a949bf4579df22fd42ce0088a364e3f69a7468a3c12919be792ef5",
                "size": 86086304
            },
            "DiffID": "sha256:aa8400e5c7087ed0c6759fc10335e35496bb1ad000a339863d0ab7f0edefdab7"
        },
        {
            "Descriptor": {
                "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                "digest": "sha256:7c19e8c974119414f370ad5ff4985359d54e9b0224e5c4217c257d7029c0d284",
                "size": 126
            },
            "DiffID": "sha256:884387fed6be6e75a6db24650f75a379b4b67bf6bbf8d2296645dcefbd520189"
        },
        {
            "Descriptor": {
                "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                "digest": "sha256:5822965cb9903f1f1278481bb61b99d05619c09a072f17ab7bbd70fb564eb909",
                "size": 258
            },
            "DiffID": "sha256:45f985075ccd50b014cb711d10df9d4b5182ed7cffda6b9edbdef049feba9707"
        },
        {
            "Descriptor": {
                "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                "digest": "sha256:b6098a6fc792a706fb2c030bee670de5478e5b8d1853db77dc16b2caa17c977a",
                "size": 128385396
            },
            "DiffID": "sha256:2d2690487bd70ecd47e906f7b790ad59ef1f79af05b5bcc1043f61e21a1ce3e0"
        },
        {
            "Descriptor": {
                "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                "digest": "sha256:fbbe79194885e8cfd49211b116e106948904e6f00a2a4e120ba9dbc7d272a3d7",
                "size": 5572413
            },
            "DiffID": "sha256:c0374622e8bed0de88da739f29b64cddad53d12ab1480564ab5066b84925aa96"
        },
        {
            "Descriptor": {
                "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                "digest": "sha256:7eb215d85b046cfc79fc67639d8d9c6849e05a5d96bf0a18b0d423d725524a78",
                "size": 5195398
            },
            "DiffID": "sha256:b91352ef454a3af056f15022ae350326d22f7ea7e972ecfb1b0c8db51b91cbb4"
        },
        {
            "Descriptor": {
                "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                "digest": "sha256:6764dbabc86abdc94a59eb69db93c69b41f268ecc5fc919c990b24a7d2dff54b",
                "size": 335
            },
            "DiffID": "sha256:0b47e97c97aecd3408508cb8a4f00d0bc6706a88209a13c60d091076a9225130"
        },
        {
            "Descriptor": {
                "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                "digest": "sha256:b719fa17491eade294458b2abcdd759d1a64452decf207e383f362e27a30b42e",
                "size": 221
            },
            "DiffID": "sha256:479174c181495b2326cae4020d28773a8ec86981c3ed20a94e88050d44fcb45d"
        }
    ],
    "envs": [
        "PATH=/ide/bin:/usr/bin",
        "EDITOR=gp open -w",
        "GIT_EDITOR=gp open -w",
        "VISUAL=gp open -w"
    ]
}